### ISSUE TITLE
Equity like share vote 

### ISSUE THREAD

**Issue by ckpringle** [vegaprotocol/frontend-monorepo#1512]
## Story

As a liquidity provider
I want to be able to take part in and see the results of equity like share votes for change market proposals (to progress the proposal even if the required majority has not been met)
So that I can take part in and see the results of equity like share votes rather than rely on using the API directly

## Acceptance Criteria

- [ ] I can see on market change proposals (proposal form and proposal details page) my equity like share and how that relates to the vote (Equity like share is a network param - governance.proposal.updateMarket.minProposerEquityLikeShare )
- [ ] I can see LP votes on a proposal details page (market change only) - we'll need to work out if this is a second vote, that can only occur if the first fails or if the two run concurrently
- [ ] As an LP I can vote on an equity like share vote 
- [ ] I can understand the difference between the regular vote and equity like share vote i.e. we've explained it
- [ ] I can see which of the vote types is the one that will be applied on the proposal details page (Token or LP is set to be the decisive one).

## Tasks

- [x] Explore and sketch
- [x] Team and stakeholder review - https://vegaprotocol.slack.com/archives/C038EB6V2RE/p1664793351695239
- [x] Visual Design (if needed)
- [x] Team review
- [x] Develop

## Sketch

![Image](https://user-images.githubusercontent.com/13255539/193780859-8248ac5d-7e58-4db2-a15e-7c4126d54c56.png)

To be clear: there is only one Vote method. If you have EQ like share it'll count that. There are not different vote methods for Tokens and QL like share.
Here is what I think the minimum change could be.

1. add another tally to show the EQ like vote status (this one doesn't need a count, only %)
2. re-label the existing vote tally as "token vote" not just "votes"
3. add some logic to show what way the vote will pass (if it will)
4. add the detail for the liquidity vote
5. more re-labelling to be clear this is token not liquidity vote detail
6. remove the details of parties and tokens (this is displayed more efficiently in the visualisation
- The re-labelling of "Votes" to "Token Votes" could be done across the board
- remove the countdown and "set to pass/fail" from each tally. There is a time stamp at the top. The set to pass fail is covered by 3.
- add senctence under the two vote tallies 

The way this falls short in that it doesn't tell you what your EQ like share is before or after you vote (it does show your tokens but only in the side bar).

## Additional details / background info

From Michael 

- on market change proposals: show your equity like share (liquidity provider vote, AKA LP) - This is not included in the design above and could be another ticket (done as part of vote re-factor?)
- Find out from @EVODelavega if you submit two votes, one for VEGA and a different for LP? - there is one vote. not
- Figma file  https://www.figma.com/file/F995ESZ0QMpTBuQWGAgLRz/Design-System?node-id=1718%3A70064 (either token or LP) (but this is WPI for a design that require more drastic layout changes. Scope of this ticket is the simplest solution.

**Comment by pennyandrews** [comment]
@ckpringle Dex identified the following related tickets. Would be great to go through these and consolidate / refine / etc. 

https://github.com/vegaprotocol/frontend-monorepo/issues/1369
https://github.com/vegaprotocol/frontend-monorepo/issues/1370
https://github.com/vegaprotocol/frontend-monorepo/issues/1110
https://github.com/vegaprotocol/frontend-monorepo/issues/1109

**Comment by pennyandrews** [comment]
David Siska was keen to chat about this before we started work on it

**Comment by campbellssource** [comment]
To get your EQlike share
```
{
  market(id:"c97538009e19f810be9631bab5a98c4a316f0d63fb9eaa7e9910c58379a61f5a"){
    data{
      liquidityProviderFeeShare{
        party{id}
        equityLikeShare
      }
    }
  }
}
```
then find the connected party in the provisions. I can't find a way in the API to get this directly.
a party with 100% EQ like share returns a `"equityLikeShare": "1"`

**Comment by campbellssource** [comment]
and to show the current vote status:
```
{
  proposal(id:""){
    votes{
      yes{
        totalEquityLikeShareWeight
      }
      no{
        totalEquityLikeShareWeight
      }
    }
  }
}
```

**Comment by jtsang586** [comment]
Hey @ckpringle, could you please confirm that the following are now removed by design? 

- Details page no longer showing number of days left to vote

- No longer showing proposal set to fail after it's just been submitted

**Comment by campbellssource** [comment]
days left to vote would be good to see still (could just be a key value table below) but was removed by design from the bar chart area under "votes" header.
@jtsang586 I'm not sure what you mean about showing set to fail. but technically if no-one has voted yet, it would fail (assuming no-one else votes, which is generally what the page does).

**Comment by dexturr** [comment]
@jtsang586 This is being reintroduced in #1820 so I suggest we close this and track there
